internal/app/config.go

```diff
@@ -51,6 +52,15 @@ type ConfigCluster struct {
 	Location string `yaml:"location"`
 }
 
+type ConfigOutputs struct {
+	PubSub ConfigPubSub `yaml:"pubsub"`
+}
+
+type ConfigPubSub struct {
+	Project string `yaml:"project"`
+	Topic   string `yaml:"topic"`
```

Comment:
It will be nice to add validation of outputs config i.e. 
`func validateOutputsConfig(outputs []ConfigOutputs) []error` and wire it inside `ValidateClusterReviewConfig`. 

Without it, it is possible to use pubsub output i.e. with an empty project.

Reply:
Added the code for validation.